ui/shared/components/panel/variants/Pathogenicity.jsx

```diff
@@ -1,12 +1,23 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Label, Rating } from 'semantic-ui-react'
+import styled from 'styled-components'
+import { Label, Icon } from 'semantic-ui-react'
 
 import { CLINSIG_SEVERITY } from '../../../utils/constants'
 import { snakecaseToTitlecase } from '../../../utils/stringUtils'
 import { HorizontalSpacer } from '../../Spacers'
 
 
+const GrayStarIcon = styled(Icon).attrs({ name: 'star' })`
+  color: #D5D5D5;
+  margin: 0em 0.3em 0em 0em !important;
+`
+
+const FilledInStarIcon = styled(Icon).attrs({ name: 'star' })`
+  color: #FFB70A;
+  margin: 0em 0.3em 0em 0em !important;
+`
+
```

Comment:
Icons support a color prop, so you could just use the builtin `<Icon name="star" color="yellow/grey" />` which I think would be clearer here. There is also a `fitted` prop if you need to decrease the margins

Reply:
oh yeah, thanks

Reply:
turns out fitted doesn't reduce margin enough and yellow/grey colors don't look right